lib/Dialect/SPIRV/Serialization/Deserializer.cpp

```diff
@@ -207,6 +207,9 @@ class Deserializer {
   // Result <id> to decorations mapping.
   DenseMap<uint32_t, NamedAttributeList> decorations;
 
+  // Result <id> to type decorations.
+  DenseMap<uint32_t, uint64_t> typeDecorations;
```

Comment:
I think it might be better to make this DenseMap<uint32_t, uint32_t> cause it will be interpreted to different types later.

Reply:
Thanks, I'll update it.